.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
I would suggest pointing people to https://github.com/gleam-lang/gleam/discussions before opening issues related to questions.

Reply:
Hope this is good now as it is (changed a bit)